### PR TITLE
Move project version to CMakeLists

### PR DESCRIPTION
**Pull request by chivay**

### CODE REVIEW

**Comment by msm-code** [comment]
Can you rebase this change onto the current master? :thinking: So we'll know if it still builds (it does, I've checked by hand) and won't have to resolve conflicts

File: Version.h.in [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#pragma once
```
Comment (msm-code) [inline Version.h.in:1]: Rebase onto master to solve merge conflicts

File: Version.h [hunk 2]
```diff
@@ -1,5 +0,0 @@
-#pragma once
-
-#include <string>
-
-constexpr std::string_view ursadb_version = "1.2.0";
```
Comment (msm-code) [inline Version.h:5]: also I think current master's version is 1.3.0

**Comment by chivay** [comment]
Multiple remotes are hard :seal:

**Comment by msm-code** [comment]
:sunglasses:

**Pull request merged**